### DISCUSSION TITLE
Go Modules path changed

### DISCUSSION THREAD

**Discussion by cloverzrg** [qax-os/excelize#1626] [General]
github.com/360EntSecGroup-Skylar/excelize/v2 --> github.com/xuri/excelize/v2

**Comment by xuri** [comment]
I'm sorry for bringing you confusion, this duplicated with #973, #1020 and #1079. For some reasons, I must accept the organization name change, so I decided to use the original module name: `github.com/xuri/excelize/v2`, and that path won't be change in the future. This module path change has been take effect since already released verion 2.4.1, and all breaking changes has been notation on the [release notes](https://xuri.me/excelize/zh-hans/releases/v2.4.1.html).
